### PR TITLE
Refinement poc copy mechanism

### PR DESCRIPTION
**Pull request by nbittich**
OP-1285

### CODE REVIEW

File: app/components/contact-details.hbs [hunk 1]
```diff
@@ -1,121 +1,165 @@
...
-          <th>Selecteer</th>
-          <th></th>
-        </c.header>
-        <c.body as |contact|>
-          <td>
-            {{contact.primaryContact.contactAddress.street}}
+      <:title>Contactgegevens</:title>
+    </Position::ContactEditCard>
+  </form>
+{{/if}}
```
Comment (claire-lovisa) [inline app/components/contact-details.hbs:20]: It could be simplified by using a `{{#if} / {{else}} / {{/if}}` no ?

Comment (nbittich) [reply app/components/contact-details.hbs]: indeed! I changed it to if/else where applicable, thanks!

**Review by claire-lovisa — CHANGES_REQUESTED** [review]
While testing I noticed two issues : 
- Sometimes I get `TypeError: this.args.contact.primaryContact is undefined` when trying to add new contact info. For ex. in my latest try I went to http://localhost:4200/bestuurseenheden/cb69c277a4021bf71a8c7cacc22bd9bc/bedienaren/nieuwe-bedienaar , took a person who already had a contact info, entered the address ` Ezaart 1, 2400 Mol` and when I tried to click `Selecteren` I had this error
- Clicking on bewerk for a mandataris and arriving on page "Bewerk mandataris" triggers of bunch of deprecation warnings such as `You attempted to update `length` on `<(unknown):ember695>`, but it had already been used previously in the same computation.`. Same on the `Voeg mandataris toe` page. I didn't see issues in the behaviour, but maybe it would be worth trying to solve them ?

**Comment by nbittich** [comment]
> While testing I noticed two issues :
> 
>     * Sometimes I get `TypeError: this.args.contact.primaryContact is undefined` when trying to add new contact info. For ex. in my latest try I went to http://localhost:4200/bestuurseenheden/cb69c277a4021bf71a8c7cacc22bd9bc/bedienaren/nieuwe-bedienaar , took a person who already had a contact info, entered the address ` Ezaart 1, 2400 Mol` and when I tried to click `Selecteren` I had this error
> 
>     * Clicking on bewerk for a mandataris and arriving on page "Bewerk mandataris" triggers of bunch of deprecation warnings such as `You attempted to update `length`on`<(unknown):ember695>`, but it had already been used previously in the same computation.`. Same on the `Voeg mandataris toe` page. I didn't see issues in the behaviour, but maybe it would be worth trying to solve them ?

thank you! both should be fixed in the last commits

**Review by claire-lovisa — APPROVED** [review]
Perfect ! :dancing_women:

**Pull request merged**